astropy/stats/funcs.py

```diff
@@ -1460,7 +1534,9 @@ def kuiper_two(data1, data2):
     return D, kuiper_false_positive_probability(D, Ne)
 
 
-def fold_intervals(intervals):
+def fold_intervals(
+    intervals: list[tuple[float, float, float]],
+) -> tuple[NDArray, NDArray]:
```

Comment:
Flagging a few stray `NDArray`, here and below.

Reply:
Thanks for catching these! Fixed in [803c421](https://github.com/astropy/astropy/pull/16562/commits/803c4213c2ff1482eb9670f0e06e57a01d6b1f8d)